src/site/stages/build/plugins/add-id-to-subheadings.js

```diff
@@ -36,15 +30,24 @@ function generateHeadingIds() {
       if (fileName.endsWith('html')) {
         const { dom } = file;
         const tableOfContents = dom('#table-of-contents ul');
-        dom('h2, h3').each((i, el) => {
+
+        const headingOptions = {
+          previousHeadings: [],
+          previousId: 0,
+          getHeadingId() {
+            return ++this.previousId;
```

Comment:
This is cool! 👍

Reply:
Ahh ya, hmm well alternatively you could create a variable outside the scope of both of these functions like `let previousId = 0;` and increment that :)

Reply:
That's similar to how it was before with a variable at the top of the file, but then the integer will be incremented globally again, instead of only in the context of the current page

Reply:
Oh! I missed that that's how it was previously implemented. I suppose after the `dom('h1', 'h3').each` we could also reset that variable back to `0`, but I like your solution as well.